package.json

```diff
@@ -15,7 +15,7 @@
     "lint:eslint-docs": "npm run update:eslint-docs -- --check",
     "lint:js": "eslint .",
     "pretest": "mkdir -p node_modules/ && ln -fs $(pwd) node_modules/",
-    "test": "npm run eslint-check && npm run lint && mocha tests/**/*.js tests/",
+    "test": "npm run eslint-check && npm run lint && mocha tests/**/*.js tests/**/*.mjs",
```

Comment:
The `mjs` tests aren't actually running since [this update](https://github.com/github/eslint-plugin-github/commit/5c76fdec19b2f9cbed3856f6fbdf59fdc85ad7ff).

Reply:
> This library seems to be using a mix of module JS and common JS now

I wonder if that's why? 🫤